test/index.js

```diff
@@ -1,53 +1,68 @@
 const path = require('path')
 const tape = require('tape')
 
-const { Diorama, tapeExecutor, backwardCompatibilityMiddleware } = require('@holochain/diorama')
+const { Orchestrator, Config, tapeExecutor } = require('@holochain/try-o-rama')
 
 process.on('unhandledRejection', error => {
   // Will print "unhandledRejection err is not defined"
   console.error('got unhandledRejection:', error);
 });
 
 const dnaPath = path.join(__dirname, "../dist/passthrough-dna.dna.json")
-const dna = Diorama.dna(dnaPath, 'passthrough-dna')
-
-const diorama = new Diorama({
-  instances: {
-    alice: dna,
-    bob: dna,
-  },
-  bridges: [],
-  debugLog: false,
-  executor: tapeExecutor(require('tape')),
-  middleware: backwardCompatibilityMiddleware,
+const dna = Config.dna(dnaPath, 'passthrough-dna')
+const config = {
+  instances: { app: dna }
+}
+
+var transport_config = {
+    type: 'sim1h',
+    dynamo_url: "http://localhost:8000"
+}
+
+if (process.env.HC_TRANSPORT_CONFIG) {
+    transport_config=require(process.env.HC_TRANSPORT_CONFIG)
+}
```

Comment:
@zippy this feels too open. I can't think of an explicit way to exploit it, but it feels kinda wrong to be able to run an arbitrary (existing) file just by setting an env var (which could easily be tucked away in some startup script and unnoticed by the user). Yet I can't say for sure that this is not OK. Quick thoughts on it before merging?

Reply:
Ok, I'll edit to use: https://www.npmjs.com/package/config